src/io/flutter/bazel/PluginConfig.java

```diff
@@ -90,6 +96,21 @@ PluginConfig load(@NotNull VirtualFile file) {
     return ApplicationManager.getApplication().runReadAction(readAction);
   }
 
+  @VisibleForTesting
+  public static PluginConfig forTest(
+    @Nullable String daemonScript,
+    @Nullable String doctorScript,
+    @Nullable String launchScript,
+    @Nullable String testScript
+  ) {
+    final Fields fields = new Fields();
```

Comment:
Should we add an additional ctor to `Fields` here? It would mean less assignment below.